com.chartboost.helium/Runtime/Platforms/HeliumIOS.cs

```diff
@@ -11,30 +11,29 @@ public sealed class HeliumIOS : HeliumExternal
     {
         #region Objective-C Extern Members
         // callback definitions for objective-c layer
-        private delegate void ExternHeliumEvent(int errorCode, string errorDescription);
+        private delegate void ExternHeliumEvent(string error);
         private delegate void ExternHeliumILRDEvent(string impressionDataJson);
         private delegate void ExternHeliumPartnerInitializationDataEvent(string partnerInitializationData);
-        private delegate void ExternHeliumPlacementEvent(string placementName, int errorCode, string errorDescription);
+        private delegate void ExternHeliumPlacementEvent(string placementName, string error);
+        private delegate void ExternHeliumPlacementLoadEvent(string placementName, string loadId, string error);
         private delegate void ExternHeliumWinBidEvent(string placementName, string auctionId, string partnerId, double price);
-        private delegate void ExternHeliumRewardEvent(string placementName, int reward);
 
         [DllImport("__Internal")]
         private static extern void _setLifeCycleCallbacks(ExternHeliumEvent DidStartCallback,
             ExternHeliumILRDEvent DidReceiveILRDCallback, ExternHeliumPartnerInitializationDataEvent DidReceivePartnerInitializationDataCallback);
 
         [DllImport("__Internal")]
-        private static extern void _setInterstitialCallbacks(ExternHeliumPlacementEvent DidLoadCallback,
-            ExternHeliumPlacementEvent DidShowCallback, ExternHeliumPlacementEvent DidClickCallback,
-            ExternHeliumPlacementEvent DidCloseCallback, ExternHeliumPlacementEvent DidRecordImpression, ExternHeliumWinBidEvent DidWinBidCallback);
+        private static extern void _setInterstitialCallbacks(ExternHeliumPlacementLoadEvent DidLoadCallback,
```

Comment:
I assume you wanted to go with `External`
```suggestion
        private static extern void _setInterstitialCallbacks(ExternalHeliumPlacementLoadEvent DidLoadCallback,
```